packages/email-templates/package.json

```diff
@@ -11,9 +11,9 @@
     "handlebars": "4.7.7"
   },
   "devDependencies": {
-    "@vitest/coverage-c8": "0.29.7",
+    "@vitest/coverage-c8": "0.28.3",
```

Comment:
Downgrading as we have that annoying error
```
TypeError: The "path" argument must be of type string. Received an instance of Array
```

Reply:
Aldo found out that when the package for email templates was added, the tests were not added to CI: https://github.com/unlock-protocol/unlock/issues/11384